assets/js/googlesitekit/data/utils.js

```diff
@@ -113,12 +151,40 @@ export const collectReducers = ( ...args ) => {
 	};
 };
 
+/**
+ * Collects all resolvers.
+ *
+ * @since n.e.x.t
+ * @param {Object} ...args A list of objects, each containing their own resolvers.
```

Comment:
According to JSdoc, these should be like so
```suggestion
 * @param {...Object} args A list of objects, each containing their own resolvers.
```